### ISSUE TITLE
Can the pretrained models be used for commercial purposes?

### ISSUE THREAD

**Issue by haibowang1427** [google/mediapipe#2595]
Per mediapipe's license, is it 👌  to use mediapipe and the pretrained models (especially facemesh) for commercial purpose? Is there any privacy concerns on this regard? 

Thank you for any comments.

**Comment by Pandinosaurus** [comment]
**Disclaimer**: 
I am not a lawyer, I am not a googler ; I just read the license.

**About Facemesh model**
According to the current model card of Facemesh on 1st October 2021 ([official list of model cards used in mediapipe](https://google.github.io/mediapipe/solutions/models.html), [direct link to the official pdf](https://drive.google.com/file/d/1QvwWNfFoweGVjsXF3DXzcrCnz-mx-Lha/edit)) current Facemesh model ([face_landmark.tflite](https://github.com/google/mediapipe/blob/master/mediapipe/modules/face_landmark/face_landmark.tflite))  used in current mediapipe master branch (commit `137e1cc`) has been released under Apache License Version 2.0. If Google respects the license terms, nothing prevents you from using the current version of Facemesh model in a commercial application. Whether the License should be changed in the future, new license should not be applicable for the current model if you used it under previous license terms. 

Relevant quote from Apache License Version 2.0 :
```
   2. Grant of Copyright License. Subject to the terms and conditions of
      this License, each Contributor hereby grants to You a **perpetual**,
      worldwide, non-exclusive, no-charge, royalty-free, **irrevocable**
      copyright license to reproduce, prepare Derivative Works of,
      publicly display, publicly perform, sublicense, and distribute the
      Work and such Derivative Works in Source or Object form.
```

**Comment by sgowroji** [comment]
Hi @haibowang1427 , You can refer the https://github.com/google/mediapipe/issues/1290#issuecomment-727150276 for your query .  Thanks!
